src/custom_exception.py

```diff
@@ -0,0 +1,3 @@
+class NoUserInDatabase(Exception):
```

Comment:
@rajan10 , what if you want to handle a case where there is no task in database, will you create a new class with name `NoTaskInDatabase` ??

I don't wanna be doing that, instead create a generalized class which can be used when no object is found in the database.